### PR TITLE
[hi] Localize content/en/docs/reference/glossary/cidr.md

### PR DESCRIPTION
**Pull request by taniaduggal**
Signed-off-by: Tania Duggal <103496926+taniaduggal@users.noreply.github.com>
Fixes #35600

### CODE REVIEW

**Comment by taniaduggal** [comment]
@mittalyashu please review:)

File: content/hi/docs/reference/glossary/cidr.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+   CIDR IP पतों के ब्लॉक का वर्णन करने के लिए एक संकेतन है और विभिन्न नेटवर्किंग कॉन्फ़िगरेशन में इसका भारी उपयोग किया जाता है।
+aka:
+tags:
+- networking
+---
+CIDR(क्लासलेस इंटर-डोमेन रूटिंग) IP पतों के ब्लॉक का वर्णन करने के लिए एक संकेतन है और विभिन्न नेटवर्किंग कॉन्फ़िगरेशन में इसका भारी उपयोग किया जाता है।
+
+<!--more-->
+
+कुबेरनेट्स के संदर्भ में, प्रत्येक {{<glossary_tooltip text="नोड" term_id="node" >}} को आरंभिक पते के माध्यम से IP पतों की एक श्रृंखला और CIDR का उपयोग करके एक सबनेट मास्क सौंपा गया है। यह नोड्स को प्रत्येक {{<glossary_tooltip text="पॉड" term_id="pod" >}} को एक अद्वितीय IP पता निर्दिष्ट करने की अनुमति देता है। हालांकि मूल रूप से IPv4 के लिए एक अवधारणा है, CIDR को भी IPv6 को शामिल करने के लिए विस्तारित किया गया है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/cidr.md:16]:
```suggestion
कुबेरनेट्स के संदर्भ में, प्रत्येक {{<glossary_tooltip text="नोड" term_id="node" >}} को आरंभिक पते के माध्यम से IP पतों की एक श्रृंखला और सीआईडीआर का उपयोग करके एक सबनेट मास्क सौंपा गया है। यह प्रत्येक {{<glossary_tooltip text="पॉड" term_id="pod" >}} को एक अद्वितीय IP पता निर्दिष्ट करने की अनुमति नोड्स को देता है। हालाँकि यह मूल रूप से IPv4 के लिए एक अवधारणा थी, IPv6 को शामिल करने के लिए CIDR का विस्तार किया गया है |
```

**Review by divya-mohan0209 — DISMISSED** [review]
Some nits.

**Comment by divya-mohan0209** [comment]
Hello @taniaduggal - please squash commits so that we can proceed with merging the PR.

@bishal7679 @Babapool @Garima-Negi @verma-kunal @anubha-v-ardhan : PTAL & confirm if any changes are required.

**Review by Babapool — COMMENTED** [review]
The content looks okay to me. However in same places CIDR is written as `CIDR` and in some places it is `सीआईडीआर`, so to maintain consistency in the glossary, in which way should CIDR be used here, Thoughts? I would love to hear other's opinions as well. Thanks

**Comment by divya-mohan0209** [comment]
Closing this PR since a duplicate one is open #38337 
/close

**Pull request closed without merging**